### PR TITLE
feature/#6902-allocated_claims_api

### PR DESCRIPTION
**Pull request by reissjohnson**
connects #6902

### CODE REVIEW

File: source/includes/api/performance/_allocated_claims.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+## Allocated Claims
+
+### Create Allocated Claims
+
+Endpoint to create allocated claims against specific promotion. Claims are created by using the `data_field` names from above as key/value pairs. Your request must include user group name or company name, depending on the promotion configuration.
```
Comment (rjpaskin) [inline source/includes/api/performance/_allocated_claims.md]: "user group name" -> "user group id"?
"company name" -> "company id"?

File: source/includes/api/performance/_allocated_claims.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+Attribute | Type | Info
+--------- | ---- | ----
+promotion\_id | `integer` | The ID of the promotion claim is for.
+sale\_date | `date` | The date when this piece of data was created. Mandatory. 
+product\_or\_activity\_ref | `string` | The SKU for the performance product to claim against
+quantity | `integer` | The quantity of products sold. Mandatory.
+user\_group\_id | `integer` | ID of the user group claim relates to (either this or company_id needs to be supplied)
+company\_id | `integer` | ID of the MyRewards company claim related to (either this or user_group_id needs to be supplied)
+
+You will also need to provide any custom data fields as extra keys for each piece of claim data in snake case format which is provided in the name field for the GET List Data Fields for a Promotion.
```
Comment (rjpaskin) [inline source/includes/api/performance/_allocated_claims.md]: Link to "GET List Data Fields for a Promotion" section?

File: source/includes/api/performance/_allocated_claims.md (hunk 1 above)
Comment (reissjohnson) [reply source/includes/api/performance/_allocated_claims.md]: Good spot, sorry this was pulled straight over from the ticket description.

File: source/includes/api/performance/_allocated_claims.md [hunk 3]
```diff
@@ -78,3 +78,6 @@ Eg:
 Attribute | Type | Info
 --------- | ---- | ----
 some\_custom\_field | `custom field type` | A custom answer for this field
+
+These data fields can be obtained from the following endpoint...
+[`GET /api/v2/performance/promotions/1/data_fields`](/#list-all-data-fields)
```
Comment (rjpaskin) [inline source/includes/api/performance/_allocated_claims.md]: 1. Don't need the `/` in the URL?
2. Feels a bit redundant having the link here, transform "GET List Data Fields" in the paragraph above to the link?

**Pull request merged**